usage/usage_test.go

```diff
@@ -78,21 +78,22 @@ func idFor(org int, metric, unit, mtype string, tags []string, interval uint32)
 
 func assertLen(epoch int, aggmetrics *FakeAggMetrics, l int, t *testing.T) {
 	aggmetrics.Lock()
-	if len(aggmetrics.Metrics) != l {
+	// temporarily commented until flakiness is fixed
```

Comment:
since you rebased on top of code that disables the usage tests, this change is not needed anymore and should be removed.